client/src/profile/ProfileContainer.js

```diff
@@ -95,16 +95,18 @@ class ProfileContainer extends React.Component {
 }
 
 ProfileContainer.propTypes = {
-  profile: PropTypes.object,
-  user: PropTypes.object.isRequired,
+  accessToken: PropTypes.string,
+  email: PropTypes.string,
```

Comment:
If someone directly accesses the /profile page there wouldn't necessarily be an accessToken, but they would be redirected